### ISSUE TITLE
Save analyzer issue: unrecognized property type `NameProperty`

### ISSUE THREAD

**Issue by nicosemp** [spicyboys/drg-completionist#117]
I have an issue with my save and it's probably caused by the fact that I started playing through the Xbox Windows app, even though I moved the save over to Steam a couple of years ago and now use it there only.

The error I get in console is:

```
context property map -> property -> struct property -> property map -> property -> array property -> struct array -> property map -> property
[External { parser: MapRes, message: "unrecognized property type `NameProperty`" }, Parser(ManyTill), Parser(Count), Parser(ManyTill), Parser(ManyTill)]
ActionName
```

My save file: https://drive.google.com/file/d/18E5ZeRw3ojXp5lhcj_CrJCOg2N4FPHf3/view

I'm available for any other information required! Thanks

**Comment by rob0rt** [comment]
Thanks for the report, sorry for missing your comment on the other Issue. Looks like I'm just missing a `NameProperty` in the parser

https://github.com/rob0rt/drg-save-parser/blob/main/gvas_save_parser/src/parser/properties/mod.rs#L130-L141

From looking at what you shared in your report, I think this'll be a simple string property type, but I'll verify against your save file

**Comment by nicosemp** [comment]
No worries! Thank you for looking at it 🙏

**Comment by rob0rt** [comment]
Fixed and deployed as of 5.0.3, now live. Thanks the report and sorry for the trouble!

**Comment by nicosemp** [comment]
Awesome thanks! I saw you updated the design from version 5 too, congrats 🎉
